hugr-py/src/hugr/_cfg.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Sequence
+from ._hugr import Hugr, Node, Wire
+from ._dfg import DfBase, _from_base
+from ._tys import Type, FunctionType, TypeRow, Sum
+import hugr._ops as ops
+
+
+class Block(DfBase[ops.DataflowBlock]):
```

Comment:
I think `Block` needs to override `_wire_up` to allow dom edges?

Reply:
oh yes I knew I forgot something, good catch, done